3 - PB/Documentazione interna/Verbali/24-04-07/24-04-07.typ

```diff
@@ -0,0 +1,94 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "07/04/24",
+  subTitle: "Meeting di retrospettiva e pianificazione",
+  docType: "verbale",
+  authors: (
+    "Carraro Riccardo",
+  ),
+  timeStart: "10:00",
+  timeEnd: "11:00",
+);
+
+= Ordine del giorno
+- Valutazione del progresso generale;
+- Analisi retrospettiva;
+- Candidatura per sostenere il colloquio della Product Baseline;
+- Predisposizione della presentazione per primo colloquio della Product Baseline;
+- Pianificazione.
+
+= Valutaizone del progresso generale
+Lo Sprint 22 si conclude con il completamento dei task pianificati, rispecchiando quanto impostato all'interno dello Sprint Backlog. 
+
+== #adr
+In data 03/04/24 il #cardin ha espresso giudizio positivo in merito all'espansione apportata al documento di #adr, ritenendo comprese le mancanze individuate nella versione del documento consegnato in RTB, e la sua espansione soddisfacente. Tale risultato ha permesso al gruppo di procedere alla candidatura per il colloquio della Product Baseline.
```

Comment:
Toglierei il "di" prima di `#adr`

Edit: dato che è una cosa presente in tutto il verbale scrivo tutto qui. Io lo toglierei ogni volta che compare prima del nome di un documento, più che altro perché mi pare suoni meglio senza, e per coerenza con gli altri verbali/documenti, dove, per l'appunto, non viene utilizzato